tests/kola/test-coreos.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+set -ex
+
+REPO=https://github.com/coreos/coreos-assembler.git
+RHCOS_SLB_TEST_PATH=mantle/kola/tests/misc/network.go
+TESTS_LIST=(rhcos.network.multipleNics)
+TMP_COREOS_ASSEMBLER_PATH=$(mktemp -d -u -p /tmp -t coreos-assembler-XXXXXX)
+IMAGE_PATH=/tmp/rhcos-latest-image
+RHCOS_SLB_REPO_PATH=$(pwd)
+
+fetch_repo() {
+  local destination=$1
+  local url=$2
+  local commit=$3
+
+  if [ ! -d ${destination} ]; then
+    mkdir -p ${destination}
+    git clone ${url} ${destination}
+  fi
+
+  (
+    cd ${destination}
+    git reset ${commit} --hard
+  )
+}
+
+fetch_latest_rhcos_image() {
+  local image_path=$1
+  local image_url="https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/latest/"
+  local latest_image_gz=$(curl ${image_url} | grep -o rhcos-[0-9].[0-9].[0-9][0-9]-x86_64-qemu.x86_64.qcow2.gz | head -1)
+  if [[ -z "${latest_image_gz}" ]]
+  then
```

Comment:
```suggestion
  if [[ -z "${latest_image_gz}" ]]; then
```

Let's keep the script consistent

Reply:
Done